website/docs/setup/installation/manual/from-source.md.erb

```diff
@@ -67,7 +61,7 @@ The following steps should be used to compile Vector directly on Linux based sys
 
     ```bash
     mkdir -p vector && \
-      curl -sSfL --proto '=https' --tlsv1.2 https://api.github.com/repos/timberio/vector/tarball/v<%= metadata.latest_version.minor_x %> | \
```

Comment:
This was done as part of https://github.com/timberio/vector/issues/1255, @a-rodin was #1255 partially completed?

Reply:
#1255 added `.X` versions for pre-compiled packages from packages.timber.io. However, the source archive is served from GitHub, so it doesn't seem to support arbitrary redirects. However, it seems like GitHub assets support URLs like https://api.github.com/repos/timberio/vector/tarball/v0.8 which return the most recent release. I think we need to use them instead.